pre_commit/util.py

```diff
@@ -36,10 +36,10 @@
 Dumper = getattr(yaml, 'CSafeDumper', yaml.SafeDumper)
 
 
-def yaml_dump(o: Any) -> str:
-    # when python/mypy#1484 is solved, this can be `functools.partial`
```

Comment:
oh, I think the comment is valid again, I'll just fix it when I merge it don't worry